src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -152,11 +152,13 @@ public void doIndex(StaplerRequest req, StaplerResponse resp) {
     }
 
     private PullRequest getPullRequest(String payload, GitHub gh) throws IOException {
+        @SuppressWarnings("UnnecessaryLocalVariable")
         PullRequest pr = gh.parseEventPayload(new StringReader(payload), PullRequest.class);
         return pr;
     }
 
     private IssueComment getIssueComment(String payload, GitHub gh) throws IOException {
+        @SuppressWarnings("UnnecessaryLocalVariable")
```

Comment:
Restored your local variables for debugging and suppressed the warnings so it's known it's intentional.